config.yaml.example

```diff
@@ -2,6 +2,7 @@ mqtt:
   host: 192.168.1.1
   username: user
   password: password
+  topic_prefix: hostname
```

Comment:
Maybe we should also add port here, if its added in code, just to show its possible to specify it here, without reading mqtt.py code.

Reply:
@zewelor you are right